src/FixedDataTableBufferedRows.js

```diff
@@ -142,7 +144,7 @@ var FixedDataTableBufferedRows = React.createClass({
 
       this._staticRowArray[i] =
         <FixedDataTableRow
-          key={i}
+          key={rowKeyGetter(rowIndex)}
```

Comment:
@wcjordan Do you think there is any risk in this change? I remember there was discussion about indexing this by rowIndex instead of its array order in the past